packages/frontend/src/components/profile_picture.tsx

```diff
@@ -0,0 +1,71 @@
+import type { ChangeEvent } from "react";
+import { useState } from "react";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import { faPlus } from "@fortawesome/free-solid-svg-icons";
+
+interface ProfilePicturePickerProps {
+  backgroundColor?: string;
+}
+
+export function ProfilePicturePicker({
+  backgroundColor = "bg-p-grey-100",
```

Comment:
We want to pass in an `onChange: (img: string) => void` function which passes in the image, so that we can have access to it in the page